<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FigDialog</class>
 <widget class="QDialog" name="FigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Тип фигуры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="circleButton">
       <property name="text">
        <string>Круг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="triangleButton">
       <property name="text">
        <string>Треугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="squareButton">
       <property name="text">
        <string>Квадрат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="pentagonButton">
       <property name="text">
        <string>Пятиугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hexButton">
       <property name="text">
        <string>Шестиугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Размер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="figSizeSlider">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Толщина границы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="penSizelSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="brushColorButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="penColorButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Цвет грницы</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Цвет заливки</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
